docs/docs/get-started.md

```diff
@@ -48,8 +50,10 @@ Recce expects two sets of dbt [artifacts](https://docs.getdbt.com/reference/arti
 - `target-base/` - dbt artifacts for to be used as the base for the comparison e.g. production
 - `target/` - dbt artifacts for your development branch
 
-#### Generate artifacts for the `base` environment
+#### Prepare artifacts for the `base` environment
+For most data warehouses, you can download the artifacts from your main branch. You don't need to re-run the whole production in your local. 
 
+However, if you use duckdb, you need to generate the artifacts for the base environment. 
```

Comment:
Yes, it's correct as you write.
For duckdb users, they may need to provide their base env.
So they add the target for base, build/run on it, and store dbt artifacts into the `target-base` directory. (e.g. `dbt run --target prod --target-base target-base`)